hydra-configs-torchvision/hydra_configs/torchvision/transforms/transforms.py

```diff
@@ -98,13 +98,15 @@ class RandomAffineConf:
 @dataclass
 class RandomApplyConf:
     _target_: str = "torchvision.transforms.transforms.RandomApply"
+    _convert_: str = 'all'
```

Comment:
editing generated code is a no-no.
What I meant was to pass `{"_convert_" : "all"}`  in the config of the specific tests.

Reply:
Hmm ok I thought we discussed including some manual changes. Passing them as a part of the cfg doesn't seem to work.